app/forms/flood_risk_engine/steps/correspondence_contact_email_form.rb

```diff
@@ -45,29 +45,48 @@ def self.params_key
       # The helpers ; allow_blank: true, allow_nil: true  ; only seem to apply to the email_address field
       # not the confirmation, so this format prevents both format & blank messages when conf = blank
       #
-      validation :passwords_match, if: :confirmation_present? do
+      validation :passwords_match?, if: :confirmation_present? do
         # Although not specified, Rails automatically validates against a field called email_address_confirmation
```

Comment:
There is documentation of this behaviour on the long hand form of the validation `validates_confirmation_of`. Perhaps a link to [that doc](http://api.rubyonrails.org/classes/ActiveModel/Validations/HelperMethods.html#method-i-validates_confirmation_of) woud be useful in the comments